merge-release-run.js

```diff
@@ -12,7 +12,13 @@ const get = bent('json', process.env.NPM_REGISTRY_URL || 'https://registry.npmjs
 
 const event = JSON.parse(fs.readFileSync('/github/workflow/event.json').toString())
 
-let pkg = require(path.join(process.cwd(), 'package.json'))
+const deployDir = path.join(process.cwd(), process.env.DEPLOY_DIR || './')
```

Comment:
this should also use `path.normalize` to remove the dangling `./` on the default case.

Reply:
actually, it looks like `.join` does normalization now. i wonder why i thought it didn’t.